tutorials/vr/openxr/handtracking.rst

```diff
@@ -0,0 +1,121 @@
+
+.. image:: img/hand_tracking_nodes.png
+
+First we find the child node `Wrist`, underneath here there are nodes for each finger. Each node starts with the name of the finger followed by the name of the bone. The finger names are `Thumb`, `Index`, `Middle`, `Ring` and `Little`. The bone names are `Metacarpal`, `Proximal`, `Intermediate`, `Distal` and `Tip`. Ergo IndexDistal is the distal bone of the index finger.
+
+.. note::
+    The thumb is the only finger that does not have a intermediate bone! 
+
+The parent-child relationships of these nodes are important and the hand will only look correct if this structure if followed exactly. Note that adding extra nodes isn't a problem, the example scenes add a number of extra bone meshes in to complete the look of the hand. Note also that the example scenes have scripts attached to the wrist node that update the size and positions of these extra nodes. 
+
+Skeleton based hand tracking
+----------------------------
+
+The second method supported by the OpenXR plugin is exposing the bone data as a :ref:`Skeleton <class_Skeleton>` node. In this case the solution is divided in two classes, one for placing the hand in space and the second to animate the aforementioned skeleton by updating the bone poses of the skeleton.
+
+This approach allows deforming a mesh which is a visually more pleasing solution however differences in implementation between the platforms does pose some problems.
+
+.. note::
+    Microsoft has added another API to OpenXR that allows for retrieving a properly skinned hand mesh however as they are currently the only platform supporting this API it has not yet been added to the plugin.
+
+At this point in time the plugin only exposes the data as it is provided by the OpenXR runtime. The plugin has an example implementation based on meshes that Valve has made publicly available however these work most reliable when used in conjunction with SteamVR.
+These scenes are `scenes/left_hand_mesh.tscn` and `scenes/right_hand_mesh.tscn` and can be childed to the :ref:`ARVROrigin <class_ARVROrigin>` node.
+
+.. image:: img/arvr_mesh_example.png
+
+Below is an overview of the steps needed to implement your own version.
+
+.. note::
+    The best way to implement this logic is to ask an artist to model a hand in 3D software using a real hands dimentions and create an armature for the hand that follows the bone structure exactly as the OpenXR specification dictates in the image at the top of this article. When skinning special care needs to be taken keeping in mind that if full hand tracking is available, the distance between joints will be determined by the actual size of the players hand and may thus be different to the 3D model. After importing the model into Godot you can add the required scripts to make everything work.
```

Comment:
`using real hands dimensions` instead of `using a real hands dimentions`.

Reply:
cheers